images/ose-sdn.yml

```diff
@@ -23,6 +22,6 @@ from:
   builder:
   - stream: golang
   member: openshift-enterprise-base
-name: openshift/ose-node
+name: openshift/ose-sdn
 owners:
 - mfojtik@redhat.com
```

Comment:
Hm... probably the owner should be changed too. @mfojtik is probably the one who split this out of the origin repo originally, but now it should have `aos-sdn-staff@redhat.com` like `images/ose-network-controller.yml` did